WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -515,7 +507,11 @@ def export_getJobParameter(jobID, parName):
     :param str parName: one single parameter name
     """
     if gElasticJobDB:
-      return gElasticJobDB.getJobParameters(jobID, [parName])
+      res = gElasticJobDB.getJobParameters(jobID, [parName])
+      if not res['OK']:
+        return res
+      if res['Value'].get(int(jobID)):
+        return S_OK(res['Value'][jobID])
```

Comment:
```suggestion
        return S_OK(res['Value'][int(jobID)])
```
If there is a cast in the get before, there needs to be a cast here